### PR TITLE
The Dice Challenge

### PR DESCRIPTION
**Pull request by srushtikage**

### CODE REVIEW

File: blue.mp3
Comment (srushtikage) [inline blue.mp3:1]: Not a part of dice game

File: green.mp3
Comment (srushtikage) [inline green.mp3:1]: Not a part of dice game

File: red.mp3
Comment (srushtikage) [inline red.mp3:1]: Not a part of dice game

File: wrong.mp3
Comment (srushtikage) [inline wrong.mp3:1]: Not a part of dice game

File: yellow.mp3
Comment (srushtikage) [inline yellow.mp3:1]: Not a part of dice game

**Review by raj03kumar — COMMENTED** [review]
Kindly go through the guidelines Srushti and go follow code of conduct properly.

If you are new then take your time and go through the contributing.md you will get some insights.

**Pull request closed without merging**